app/services/scrapers/high_priority.rb

```diff
@@ -18,7 +18,9 @@ def perform
       bar = ProgressBar.new(cases.count)
       puts "#{cases.count} are high priority for update for #{county.name} county"
       cases.each do |case_number|
-        court_case = ::CourtCase.find_by!(county_id: @county.id, case_number: case_number)
+        court_case = ::CourtCase.find_by(county_id: @county.id, case_number: case_number)
```

Comment:
Since we have the `attr_accessor`, we don't need the `@` on the county:

```suggestion
        court_case = ::CourtCase.find_by(county_id: county.id, case_number: case_number)
```